src/Engine/Projects/ProjectModel.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Flagsmith\Engine\Projects;
+
+use Flagsmith\Concerns\HasWith;
+use Flagsmith\Concerns\JsonSerializer;
+use Flagsmith\Engine\Organisations\OrganisationModel;
+use Flagsmith\Engine\Utils\Collections\SegmentModelList;
+
+class ProjectModel
+{
+    use HasWith;
+    use JsonSerializer;
+
+    public int $id;
+    public string $name;
+    public OrganisationModel $organisation;
+    public bool $hide_disabled_flags;
```

Comment:
It seems that for the rest of the application we're standardising on camel case but we're using snake case for the attributes. Is this just because it's easier that way as the JSON data is snake_case? Are there any conventions when it comes to casing in PHP?

Reply:
The data classes have variables named in snake cases, I have tried ensuring consistency with variable names in the Engine package. This is because PHP does not provide support for JSON to PHP class marshalling. So, it helps to have consistency with JSON data names and PHP class member names.

Reply:
Gotcha ok.